TODO.org

```diff
@@ -156,19 +152,25 @@ Key VSCode plugin implementation points:
 
   https://github.com/renpy/vscode-language-renpy/blob/master/snippets/snippets.json
 
-- [ ] block folding (or just outline-minor-mode support?)
+- [ ] block folding, hideshow (or just outline-minor-mode support?)
+
+- [X] Add editor.conf to the package
```

Comment:
Are you talking about the emacs.edit.py file here or is this something else?  I can't see any .conf files.